index.js

```diff
@@ -319,10 +315,11 @@ function parsePipelineYaml({
  * Parses pipeline template configuration
  * @method parsePipelineTemplate
  * @param   {Object}          config
- * @param   {String}          config.yaml           Pipeline Template
- * @return  {Object}                                Pipeline Template
+ * @param   {String}          config.yaml             Pipeline Template
+ * @param   {TemplateFactory} config.templateFactory  Template Factory to get templates
+ * @return  {Object}                                  Pipeline Template
  */
-async function parsePipelineTemplate({ yaml }) {
+async function parsePipelineTemplate({ yaml, templateFactory }) {
```

Comment:
since this is a breaking change, can you also reflect the change to `template-validator` and `screwdriver` repo?